bids/modeling/statsmodels.py

```diff
@@ -653,6 +695,10 @@ def __init__(self, node, entities={}, collections=None, inputs=None,
         self.model_spec = SpecCls.from_df(self.data, node.model, self.metadata)
         self.contrasts = self._build_contrasts(unique_in_contrast)
 
+        if node_reports:
+            for k, v in node_report(self).items():
+                setattr(self, k, v)
```

Comment:
This feels really awkward to me. Why not

```suggestion
        self.report_ = node_report(self) if node_reports else None
```